components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/PaginatableFilterableApplicationDAO.java

```diff
@@ -46,9 +46,23 @@ ApplicationBasicInfo[] getAllPaginatedApplicationBasicInfo(int pageNumber) throw
      * @param limit  Count value.
      * @return An array of {@link ApplicationBasicInfo} instances within the limit.
      * @throws IdentityApplicationManagementException Error in retrieving basic application information.
+     * @Deprecated This is being deprecated as introducing excludeSystemPortals to exclude system portals in response
+     * use {@link PaginatableFilterableApplicationDAO#getApplicationBasicInfo(int, int, Boolean)}.
      */
     ApplicationBasicInfo[] getApplicationBasicInfo(int offset, int limit) throws IdentityApplicationManagementException;
 
+    /**
+     * Get all the basic application information based on the offset and the limit.
+     *
+     * @param offset Starting index of the count.
+     * @param limit  Count value.
+     * @param excludeSystemPortals  Exclude system portals.
```

Comment:
Check the formatting for parameter descriptions.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6026/commits/964c6d3814dba4a93591240fb7b95b4f3b9f77c3